docs/farming-&-staking/staking/operators/register-operator.mdx

```diff
@@ -159,39 +165,34 @@ Example:
 Start a node as operator on `gemini-3g` chain:
 
 :::info
-You need to wipe (`purge-chain`) and sync your node from genesis block, since you need to sync both consensus and domain chains.
+You need to **wipe** and sync your node from genesis block, since you need to sync both consensus and domain chains.
 You do not need to wipe any existing plots. 
 :::
 
 :::note
-Ensure you replace `your_domain_id` with your domain identifier in the command and `your_operator_id` with your operator_id. If your keystore is located in a different folder, adjust the `--keystore-path` accordingly. Setting `--base-path` is optional.
+Ensure you replace `your_domain_id` with your domain identifier in the command and `your_operator_id` with your operator_id.
 :::
 
 :::tip
 You can ignore setting up `your_operator_id` while you're syncing your node. Make sure to set it after syncing and registration.
 :::
 
-:::tip
-Stake Wars are using the domain **Nova** with ID **1**.
-:::
-
 <Tabs groupId="OS">
 
 <TabItem value="windows" label="🖼️ Windows" default>
 
 <CodeBlock>
-{`target/production/subspace-node \`
---chain gemini-3g \`
+{`target/production/subspace-node run \`
+--chain gemini-3h \`
 --name your_node_name \`
 --base-path your_path_to_node_data \`
 -- \`
 --domain-id your_domain_id \`
---chain gemini-3g \`
 --operator-id your_operator_id \`
---bootnodes /ip4/3.87.28.170/tcp/40333/p2p/12D3KooWGHtULvhdKMZtzigSK1438uWXPj9rBQHVzTaKMWv1WRXp \`
---bootnodes /ip4/140.82.45.36/tcp/40333/p2p/12D3KooWJxB2PVzyNqZywoCWAMLGhKr2LCbFMXbAPzUPQZon7yt1 \`
---bootnodes /ip4/167.179.116.135/tcp/40333/p2p/12D3KooWHsRae3ip2uQAUPtpMcStHGGw777SJT3nu9maFmtsGry7 \`
---listen-addr /ip4/0.0.0.0/tcp/40333 \`
+--bootstrap-nodes /ip4/3.87.28.170/tcp/40333/p2p/12D3KooWGHtULvhdKMZtzigSK1438uWXPj9rBQHVzTaKMWv1WRXp \`
+--bootstrap-nodes /ip4/140.82.45.36/tcp/40333/p2p/12D3KooWJxB2PVzyNqZywoCWAMLGhKr2LCbFMXbAPzUPQZon7yt1 \`
+--bootstrap-nodes /ip4/167.179.116.135/tcp/40333/p2p/12D3KooWHsRae3ip2uQAUPtpMcStHGGw777SJT3nu9maFmtsGry7 \`
```

Comment:
We really need to replace this mess of IP addresses with readable domain names for 3h. Also note that these can be in the chain spec and we should add them for 3h such that for domains created by us specifying bootstrap nodes will not be required, but have "advanced" section about adding these for custom domains (bootstrap nodes for which we will probably not run at all).

Reply:
Who can help us with that? Is it @DaMandal0rian?

Reply:
Yes, he should create it on infrastructure level and someone needs to update chain spec afterwards.

Reply:
@jim-counter these will be in the format:
`bootstrap-0.nova.subspace.network`
`bootstrap-1.nova.subspace.network`
`bootstrap-2.nova.subspace.network`